features/product/managing_product_association_types/deleting_multiple_product_association_types.feature

```diff
@@ -10,7 +10,7 @@ Feature: Deleting multiple product association types
         And the store has also a product association type "Accessories"
         And I am logged in as an administrator
 
-    @ui
+    @ui @api
     Scenario: Deleting multiple product association types at once
```

Comment:
We have decided to skip these features because we weren't sure how should we process multiple deletions, but thanks for your proposal! ;)